.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: release
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'flutter-tizen' }}
+    steps:
+      # Run flutter-tizen to fetch dart command in flutter repo.
+    - name: Install flutter-tizen
```

Comment:
The flutter-tizen tool is unused. Just using `subosito/flutter-action@v1` will do.

Reply:
Done.